### DISCUSSION TITLE
Epdiyv7 with ED060SC7

### DISCUSSION THREAD

**Discussion by KarmonVon** [vroland/epdiy#292] [Q&A] [answered]
I have ED060SC7 screen and Epdiyv7 board. Can Epdiyv7 support ED060SC7 screen?

**Comment by martinberlin** [comment]
Hi @KarmonVon 
Moved this to Discussions part since it’s not really an issue with the component. Has this panel 39 pins?
V7 does not have this connector. So you will have to make an adapter or use an Epaper that has one of the on board connectors.
Otherwise please post the datasheet here

**Reply by KarmonVon** [reply]
ED060SC7 screen connector is AXT334124 which has 34 pins, I haven't found any adapter board that convert AXT334124 to 34 pins on epdiyv7

**Reply by martinberlin** [reply, marked as answer]
No I think there is not. If you are interested to make one I can offer myself to review it.
Sorry but is impossible to have all connectors since they are many. So interested parties have to collaborate making adapter or modifying one of the PCBs and placing the target FPC connector in place

**Comment by martinberlin** [comment]
If you didn’t found it is because no one made it yet.
So maybe it’s the best idea to use KiCad and design it yourself. Then you can also collaborate with the hardware adapters of epdiy.

@mcer12 designed a v5 board called Inkster that has this connector: https://github.com/mcer12/Inkster-ESP32/blob/main/Resources/Inkster_v1.3_BOM.csv

what you can do is either design an adapter or just clone v7 board and add this connector in KiCad
